ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -87,14 +81,14 @@ def _check_disk_dtl(vdisk_guid):
         :param vdisk_guid: guid of existing vdisk
         :type vdisk_guid: str
         :return: tuple
-        :rtype: tuple
+        :rtype: tuple(str, str)
```

Comment:
Completely useless function and if kept, should be renamed to ```get_<something>``` because it doesn't check anything